src/components/Register.js

```diff
@@ -2,7 +2,7 @@ import React, { useState } from 'react';
 import axios from 'axios';
 import { BaseUrl } from "../consistents";
 import { Link } from 'react-router-dom';
-function Register() {
+function RGR() {
```

Comment:
The function name `Reg` is not descriptive. Consider renaming it to something more meaningful that reflects the purpose of the function, such as `Register` or `RegisterUser`.